src/Dungeon/Rooms/Cross.elm

```diff
@@ -96,7 +96,7 @@ toWallSize roomSize =
 -}
 dots : WallSize -> List Int
 dots wallSize =
-    List.map (dot wallSize) [0..3]
+    List.map (dot wallSize) <| List.range 0 3
```

Comment:
I like this quite a bit more, I think. I could never remember the special range syntax.